array/array.js

```diff
@@ -7,3 +7,31 @@
  * How do these operations compare to that of a linked list?
  * How does the time complexity of insertion and deletion compare to that of a linked list?
  */
+
+
+class Array {
+  constructor() {
+    this.array = {};
+    this.stack = [];
+  }
+
+  push(key, value) {
+    this.object[key] = value;
+    this.stack.push(key);
+  }
+  pop() {
+    const key = this.stack.pop();
+    let property = this.object[key];
+    delete this.object[key];
+    return property;
+  }
+  get(index) {
```

Comment:
You should try and finish these methods. You're on the right track Keep on going!